src/Riok.Mapperly/AnalyzerReleases.Unshipped.md

```diff
@@ -1,2 +1,8 @@
 ; Unshipped analyzer release
 ; https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+RMG049 | Mapper | Warning | DiagnosticDescriptors
```

Comment:
We don't use the unshipped file, you can add these directly to the shipped file. Also replace the notes with the description of the diagnostic (this get's reused in the [docs](https://mapperly.riok.app/docs/configuration/analyzer-diagnostics/)).

Reply:
done in [87addb2](https://github.com/riok/mapperly/pull/708/commits/87addb2875d8b341c679c1ca3bfc5dcfa6008593)